src/actions/sessionActions.js

```diff
@@ -2,6 +2,18 @@ import { SubmissionError } from 'redux-form/immutable';
 import { sessionService } from 'redux-react-session';
 
 import sessionApi from '../api/sessionApi';
+import * as types from './actionTypes';
+import history from '../utils/history';
+import routes from '../constants/routesPaths';
+
+export const burnt = () => ({
+  type: types.TOASTED,
+});
+
+const topicUpdater = topics => ({
```

Comment:
Hmm, what about `getTopicsSuccess` for the function name and `GET_TOPICS_SUCCESS` for the action type?

Then you can start using that "standard" for the rest of the application.